test/forcing/ForcingsEngineTestHelpers.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <NGenConfig.h>
+#if NGEN_WITH_MPI
+#include <mpi.h>
+#endif
+
+#include <utilities/python/InterpreterUtil.hpp>
+#include <forcing/ForcingsEngineDataProvider.hpp>
+
+struct ForcingsEngineDataProviderTest
+  : public testing::Test
+{
+
+    ForcingsEngineDataProviderTest()
+    {
+        #if NGEN_WITH_MPI
+        MPI_Comm_size(MPI_COMM_WORLD, &mpi_.size);
+        MPI_Comm_rank(MPI_COMM_WORLD, &mpi_.rank);
+        #endif
+    }
+
+    static void SetUpTestSuite() {
+        #if NGEN_WITH_MPI
+        MPI_Init(nullptr, nullptr)
+        #endif
+
+        ForcingsEngineDataProviderTest::gil_ = utils::ngenPy::InterpreterUtil::getInstance();
+
+        data_access::detail::assert_forcings_engine_requirements();
+    }
+
+    static void TearDownTestSuite() {
+        data_access::detail::ForcingsEngineStorage::instances.clear();
+
+        ForcingsEngineDataProviderTest::gil_.reset();
+
+        #if NGEN_WITH_MPI
+        PMPI_Finalize();
+        #endif
```

Comment:
I didn't consider how this would impact multiple test fixtures calling this -- need to rethink.